discopop_explorer/parallel_utils.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of the DiscoPoP software (http://www.discopop.tu-darmstadt.de)
+#
+# Copyright (c) 2020, Technische Universitaet Darmstadt, Germany
+#
+# This software may be modified and distributed under the terms of
+# the 3-Clause BSD License.  See the LICENSE file in the package base
+# directory for details.
+
+from .PETGraphX import Node, NodeID
+from typing import List, Set
+
+global_pet = None
+
+
+def pet_function_metadata_initialize_worker(pet):
+    global global_pet
+    global_pet = pet
+
+
+def pet_function_metadata_parse_func(param_tuple):
+    func_node = param_tuple
```

Comment:
is there a reason to not use param_tuple directly but instead reassign to a new variable?